cmd/mb-main.go

```diff
@@ -29,7 +29,7 @@ var (
 	mbFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "region",
-			Value: "us-east-1",
+			Value: "",
```

Comment:
Leave this as is.. since we don't expect users to set this value.. if its empty its used as `us-east-1` in MakeBucket().  Keeping it explicit serves as a way to document..